_analyzers/language-analyzers/index.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Language analyzers
+nav_order: 100
+parent: Analyzers
+has_children: true
+has_toc: false
+---
+
+# Language analyzers
+
+OpenSearch supports the following language analyzers:
+`arabic`, `armenian`, `basque`, `bengali`, `brazilian`, `bulgarian`, `catalan`, `czech`, `danish`, `dutch`, `english`, `estonian`, `finnish`, `french`, `galician`, `german`, `greek`, `hindi`, `hungarian`, `indonesian`, `irish`, `italian`, `latvian`, `lithuanian`, `norwegian`, `persian`, `portuguese`, `romanian`, `russian`, `sorani`, `spanish`, `swedish`, `turkish`, and `thai`.
+
+To use the analyzer when you map an index, specify the value within your query. For example, to map your index with the French language analyzer, specify the `french` value for the analyzer field:
+
+```json
+ "analyzer": "french"
+```
+
+#### Example request
+
+The following query specifies the `french` language analyzer for the index `my-index`:
```

Comment:
This request creates a `text` field with a `french` subfield configured with the french analyzer. I think we can keep this request but correct the description for it. Also, it would be nice to provide just a plain example with setting an analyzer on the whole index.